Makefile.am

```diff
@@ -0,0 +1,32 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+ACLOCAL_AMFLAGS = -I m4
+
+AM_CXXFLAGS = -std=c++11 -Wall -Werror
+AM_CPPFLAGS = \
+	-I$(srcdir)/common/json \
+	-I$(srcdir)/vendor/boost_1_80_0
+
+# Two daemons
+bin_PROGRAMS = men-authd men-updated
+
+# Headers for libraries and utils under common to be listed here
+COMMON_SOURCES = common/json/json.hpp \
+	common/json/impl/boost/boost_json.hpp
+
+# Libraries objects to be listed here
+noinst_LIBRARIES = \
+	libjson.a
+
+# For each library, set the source(s)
+libjson_a_SOURCES = common/json/impl/boost/boost_json.cpp
```

Comment:
Should these perhaps be sub-targets instead?

Reply:
+1

Reply:
+1

Reply:
I leave this for you to rework @vpodzime :+1: